patches/net/minecraft/world/level/block/Block.java.patch

```diff
@@ -11,7 +11,7 @@
      private final Holder.Reference<Block> builtInRegistryHolder = BuiltInRegistries.BLOCK.createIntrusiveHolder(this);
 -    public static final IdMapper<BlockState> BLOCK_STATE_REGISTRY = new IdMapper<>();
 +    @Deprecated //Forge: Do not use, use GameRegistry
```

Comment:
Should this still be deprecated?

Reply:
Im not too sure about that unfortunately